README.md

```diff
@@ -115,24 +115,25 @@ specified are for a Windows environment. They are very similar for Linux.
     ```commandline
     py -3.9 -m venv venv
     ```
-1. Activate it:
+2. Activate it:
```

Comment:
Isn't it more convenient to let markdown "take care" of the numbering as before?

Reply:
They changed by the system when I tried to add "Run `setup.py` with `dev` tag".